pkg/errors/errors.go

```diff
@@ -22,6 +22,8 @@ package errors
 import (
 	"errors"
 	"fmt"
+
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
Out of scope for this PR, but I think we use a few other things from `kerrors` pretty frequently that may be worth aliasing in this package. Can't remember what they are right now. :)

Reply:
IsNotFound and the like?

Reply:
Yeah. No need to add them now though, can follow up later.